pkg/mesh/topology.go

```diff
@@ -314,51 +336,58 @@ func (t *Topology) Conf() *wireguard.Conf {
 
 // AsPeer generates the WireGuard peer configuration for the local location of the given Topology.
 // This configuration can be used to configure this location as a peer of another WireGuard interface.
-func (t *Topology) AsPeer() *wireguard.Peer {
+func (t *Topology) AsPeer() wireguard.Peer {
```

Comment:
Should this return a pointer again?

Reply:
Up to you! I think so, that way we can avoid copies when we pass it around/use the methods on it

Reply:
I changed it back, but then callers of this func have to check if it returned nil.

Reply:
ack